spec/features/merchants/show_spec.rb

```diff
@@ -22,6 +22,74 @@
 
       expect(current_path).to eq("/merchants/#{@bike_shop.id}/items")
     end
+    describe 'as a merchant employee' do
+      it 'has button to deactive an item next to each item' do
 
+       merchant_user = create(:random_user, merchant_id: @bike_shop.id, role: 1)
+
+       item_1 = create(:random_item, merchant_id: @bike_shop.id)
+       item_2 = create(:random_item, merchant_id: @bike_shop.id)
+       item_3 = create(:random_item, merchant_id: @bike_shop.id, active?: false)
+       @bike_shop.users << merchant_user
+
+       allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(merchant_user)
+
+       visit '/merchant/items'
+
+       within "#item-#{item_1.id}" do
+         expect(page).to have_button("Deactivate")
+       end
+
+       within "#item-#{item_3.id}" do
+         expect(page).to_not have_button("Deactivate")
+       end
+
+       within "#item-#{item_2.id}" do
+         click_button "Deactivate"
+       end
+
+       expect(current_path).to eq("/merchant/items")
+       expect(page).to have_content("#{item_2.name} is deactivated")
```

Comment:
the user story says it should flash a message indicating that "this item is no longer for sale", we might want to refactor that.